site/docs/getting-started.md

```
@@ -90,29 +92,34 @@ export LAKEKEEPER__PG_ENCRYPTION_KEY="MySecretEncryptionKeyThatIBetterNotLoose"
 ```
 
 To expose Lakekeeper behind a reverse proxy, most deployments also set:
+
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)